udapi/block/read/conllu.py

```diff
@@ -86,6 +105,19 @@ def process_document(self, document):
                 logging.debug('No next bundle to process')
                 break
 
+            # Check if all lines have correct number of data fields.
+            # Skip invalid bundle otherwise.
+            raw_bundle_check = True
+            for line in raw_bundle:
+                if re_comment_like.search(line) is not None:
+                    continue
+                if len(line.split('\t')) != len(self.node_attributes):
```

Comment:
Processing time of the `ud-treebanks-v1.4/UD_Czech/cs-ud-train.conllu` file (68495 bundles):
```
master                       : 1m2.148s
zellig_haris                 : 1m2.184s
zellig_haris + strict option : 1m5.071s
```